src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -1811,7 +1811,9 @@ def onPolyModelChange(self, top, bottom):
             # PD[ratio] -> width, npts -> npts, nsigs -> nsigmas
             if model_column not in delegate.columnDict():
                 return
-            self.poly_params[parameter_name_w] = value
+            # Map the column to the poly param that was changed
+            associations = {1: "width", 4: "npts", 5: "nsigmas"}
+            self.poly_params[f"{parameter_name}.{associations.get(model_column, 1)}"] = value
```

Comment:
Shouldn't that be `.get(model_column)` ? I think we're guaranteed to get the proper column value here. Alternatively, maybe use "width" as the default, not `1`?

Reply:
Yes! I'll fix that.